lib/DDG/Publisher/Site/Duckduckgo/Root.pm

```diff
@@ -22,6 +22,7 @@ sub pages {{
 		%page_defaults,
 		no_hero_header_icon => 1,
 		js_page_type => 'About',
+		add_address => 1,
```

Comment:
nit: I think we should use a more descriptive flag here, like `footer_show_address` or `footer_has_address`